index.css

```diff
@@ -0,0 +1,22 @@
+html {
+	height: 100%;
+
+}
```

Comment:
Let's add a blank line between blocks declarations to keep your CSS readable and cleaner.